spec/models/storage_location_spec.rb

```diff
@@ -228,5 +228,28 @@
         expect(storage_location.longitude).not_to eq(nil)
       end
     end
+
+    describe "csv_export_attributes" do
+      it "returns an array of storage location attributes, followed by inventory item quantities that are sorted by alphabetized item names" do
+        item1 = create(:item, name: 'C')
+        item2 = create(:item, name: 'B')
+        item3 = create(:item, name: 'A')
+        inactive_item = create(:item, name: 'inactive item', active: false)
+        name = Faker::Company.name
```

Comment:
I know there are a lot of specs in the repo like this, but I much prefer deterministic tests that don't use random data. E.g. specify the name and you can use the name you gave it in the expectations below.